client/layout/guided-tours/config-elements/quit.js

```diff
@@ -12,14 +12,16 @@ import { translate } from 'i18n-calypso';
  * Internal dependencies
  */
 import Button from 'components/button';
+import { targetForSlug } from '../positioning';
 import contextTypes from '../context-types';
 
 export default class Quit extends Component {
 	static displayName = 'Quit';
 
 	static propTypes = {
+		click: PropTypes.bool,
 		primary: PropTypes.bool,
-		subtle: PropTypes.bool,
+		target: PropTypes.string,
```

Comment:
css selector?

Reply:
Not sure I get what you mean but this is not strictly a CSS selector.
GTs use the [`targetForSlug`](https://github.com/Automattic/wp-calypso/blob/master/client/layout/guided-tours/positioning.js#L85-L100) function that also accept `data-tip-target` attributes.

Reply:
Just noting that it was a bit ambiguous, but if it's similar to other components I suppose that it's fine.

Reply:
Yes, `target` is like a special data type in GT that we never formalized.